packages/firestore-types/index.d.ts

```diff
@@ -35,6 +35,26 @@ export interface Settings {
   host?: string;
   /** Whether to use SSL when connecting. */
   ssl?: boolean;
+
```

Comment:
So...  it turns out our types exist in 3 .d.ts files. 😦 @jshcrowthe has plans to fix this, but for now can you update packages/firebase/index.d.ts and packages/firebase/app/index.d.ts too?